bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveMultiInstanceCommandClass.java

```diff
@@ -353,34 +334,7 @@ private void handleMultiChannelCapabilityReportResponse(SerialMessage serialMess
 			}
 
 			// Add all the command classes supported by this endpoint
-			for (int i = 0; i < serialMessage.getMessagePayload().length - offset - 3; i++) {
```

Comment:
Ok - this is confusing as I was expecting a database change as per your PR description, but in reality there are a couple of changes in the same PR?